rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -196,6 +196,20 @@ void ObjectRenderer::renderInstance(InstanceObject *instance,
 		return;
 	}
 
+
+	if (instance->object->timeOn != 0 || instance->object->timeOff != 0)
```

Comment:
This shouldn't have to be checked (we set timeOn and timeOff to 0 and 24 (check again?) for normal OBJs so they are always shown).